prac_06/car.py

```diff
@@ -0,0 +1,31 @@
+"""CP1404/CP5632 Practical - Car class example."""
+
+
+class Car:
+    """Represent a Car object."""
+
+    def __init__(self, fuel=0):
+        """Initialise a Car instance.
+
+        fuel: float, one unit of fuel drives one kilometre
+        """
+        self.fuel = fuel
+        self._odometer = 0
+
```

Comment:
how will the console read this, you could try add a __str__ method for the class to be able to read as a string